config/mappings/assessment_mapping_fund_round.py

```diff
@@ -127,7 +127,7 @@
         "location": "rmBPvK",
         "asset_type": None,
         "funding_one": None,
-        "funding_two": ["qwktzL", "JizgZP"],  # only revenue funding for cyp
+        "funding_two": ["JXKUcj", "OnPeeS"],  # only revenue funding for cyp
```

Comment:
you may need to remove this `"field_type": "multiInputField",` from CYP as it's not multiInputField anymore

Reply:
good catch - perhaps we should specify on that field key, something like 'funding_field_type''